core/components/atoms/dropdown/__stories__/variants/WithErrorTempate.story.jsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { Dropdown, Heading } from '@/index';
+import { Uncontrolled, Controlled } from '../_common_/types';
+
+// CSF format story
+export const withErrorTemplate = () => {
+  const options = [];
+
+  const getSearchedOptions = (options, searchTerm) => {
+    const result = options.filter((option) => option.label.toLowerCase().includes(searchTerm.toLowerCase()));
+    return result;
+  };
```

Comment:
When `options` declared above is always empty then this function always return the empty array. There is no need to call this function when we can directly pass empty array to promise